projects/demo/src/modules/components/rating/examples/import/define-options.md

```diff
@@ -0,0 +1,19 @@
+```ts
+import {TUI_RATING_OPTIONS} from '@taiga-ui/kit';
+// ...
+
+@NgModule({
+  providers: [
+    {
+      provide: TUI_RATING_OPTIONS,
+      useValue: {
+        min: 0,
+        max: 5,
+        iconNormal: 'tuiIconStarLarge',
```

Comment:
Let's add different values here and show how to keep the rest default. Something like:
```ts
useValue: {
  ...DEFAULTS,
  max: 10,
  iconNormal: 'tuiIconHeart'
  iconFilled: 'tuiIconHeartFilled'
}
```
Check the actual icons names, I might be wrong )

Reply:
fixed